macros/contexts/contextInteger.pl

```diff
@@ -184,11 +184,12 @@ sub phi {
 	$self->Error("Cannot phi on Zero.") if $a == 0;
 	$result = $a;
 	$n      = $a;
-	for (my $i = 2; ($i**2) < $n; $i++) {
+	for (my $i = 2; ($i**2) <= $n; $i++) {
+		next unless ($n % $i == 0);
 		while ($n % $i == 0) {
 			$n      /= $i;
```

Comment:
I am not sure why the Github perltidy action is not running on this pull request.  However, if it were running it would give an error here.  This line should be
```suggestion
			$n /= $i;
```

Reply:
Good catch.  I can just commit that.

Reply:
Yeah, I thought that spacing was weird!  I didn't need to change that line to fix the code so I didn't mess with it.

Reply:
The spacing was correct when the `$result -= $result / $i` was below it inside the while loop.  When that was moved out, the spacing needed to be removed.

We have a Github workflow the runs perltidy on the code.  If perltidy changes anything, then the workflow fails.  If the workflow fails we won't merge the pull request until the issue is fixed.  Webwork2 and PG code has historically been quite messy and hard to read.  This makes it difficult to move forward with future development.  Perltidy is a nice tool that helps to prevent that mess.